### PR TITLE
Added NREL 5MW Openfast example

### PR DESCRIPTION
**Pull request by tonyinme**
Hey @jrood-nrel and @sayerhs, I am adding this pull request to start the process on getting an example that uses openfast within the repository. @sayerhs mentioned we might not be ready for this yet, but I'd like to have this here so we don't forget. Let me know what else I should do.

Thanks!
Tony

### CODE REVIEW

File: test/test_files/act_openfast_nrel5mw/fast_inp/5MW_Baseline/ServoData/Makefile [hunk 1]
```diff
@@ -0,0 +1,262 @@
+# CMAKE generated file: DO NOT EDIT!
```
Comment (psakievich) [inline test/test_files/act_openfast_nrel5mw/fast_inp/5MW_Baseline/ServoData/Makefile:1]: this file needs to be deleted

File: test/test_files/act_openfast_nrel5mw/fast_inp/5MW_Baseline/ServoData/CTestTestfile.cmake [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# CMake generated Testfile for
```
Comment (psakievich) [inline test/test_files/act_openfast_nrel5mw/fast_inp/5MW_Baseline/ServoData/CTestTestfile.cmake:1]: this file needs to remove hard coded paths to @tonyinme 's home directory

File: test/test_files/act_openfast_nrel5mw/fast_inp/5MW_Baseline/ServoData/CMakeFiles/progress.marks [hunk 3]
```diff
@@ -0,0 +1 @@
+0
```
Comment (psakievich) [inline test/test_files/act_openfast_nrel5mw/fast_inp/5MW_Baseline/ServoData/CMakeFiles/progress.marks:1]: probably delete this one too

File: test/test_files/act_openfast_nrel5mw/fast_inp/5MW_Baseline/ServoData/CMakeFiles/DISCON.dir/DependInfo.cmake [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# The set of languages for which implicit dependencies are needed:
+set(CMAKE_DEPENDS_LANGUAGES
+  "Fortran"
+  )
+# The set of files for implicit dependencies of each language:
+set(CMAKE_DEPENDS_CHECK_Fortran
+  "/home/lmartine/nalu-wind/reg_tests/test_files/nrel5MWactuatorLine/5MW_Baseline/ServoData/DISCON.F90" "/home/lmartine/nalu-wind/build/reg_tests/test_files/nrel5MWactuatorLine/5MW_Baseline/ServoData/CMakeFiles/DISCON.dir/DISCON.F90.o"
```
Comment (psakievich) [inline test/test_files/act_openfast_nrel5mw/fast_inp/5MW_Baseline/ServoData/CMakeFiles/DISCON.dir/DependInfo.cmake:7]: these hard coded paths aren't mergable

File: test/test_files/act_openfast_nrel5mw/fast_inp/5MW_Baseline/ServoData/CMakeFiles/CMakeDirectoryInformation.cmake [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# CMAKE generated file: DO NOT EDIT!
+# Generated by "Unix Makefiles" Generator, CMake Version 3.17
+
+# Relative path conversion top directories.
+set(CMAKE_RELATIVE_PATH_TOP_SOURCE "/home/lmartine/nalu-wind")
```
Comment (psakievich) [inline test/test_files/act_openfast_nrel5mw/fast_inp/5MW_Baseline/ServoData/CMakeFiles/CMakeDirectoryInformation.cmake:5]: these hard coded paths aren't mergable

**Review by psakievich — CHANGES_REQUESTED** [review]
I marked a few of them, but there are multiple hard absolute coded paths in here that are not suitable for merging. There are also some binary files that should be removed.  I think what we need to do here is create a new repository specifically for managing this test that will compile the servo library and make that a submodule to amr-wind and nalu-wind

**Comment by tonyinme** [comment]
Thanks @psakievich , I deleted the extra files.

**Pull request closed without merging**